src/components/PlayLists/index.jsx

```diff
@@ -1,7 +1,14 @@
+import PlayListsItem from "../PlayListItem/index.jsx";
+import playlistsData from "../../../assests/json/playlists.json";
 import "./styles.css";
 
 const PlayLists = () => {
-  return <p>Playlists</p>;
+  const playlistsItems = playlistsData.playlists.items;
+  const playlists = playlistsItems.map((playlistsItem) => (
+    <PlayListsItem details={playlistsItem} />
+  ));
+
+  return <ul class="playlists">{playlists}</ul>;
```

Comment:
No need to define a variable. Let the ui part remain in the return unless it's necessary.
```suggestion
  return <ul class="playlists">{playlistsItems.map((playlistsItem) => (
    <PlayListsItem details={playlistsItem} />
  ))}</ul>;
```

Reply:
doneeeeeee